configuration/frontend/base-config.json

```diff
@@ -55,10 +55,15 @@
   "@openmrs/esm-login-app": {
     "logo": {
       "src": "/openmrs/spa/site/pih-logo.png",
-      "alt": "Partners in Health"
+      "alt": "Partners In Health"
     },
     "links": {
       "loginSuccess": "${openmrsBase}/pihcore/home.page"
+    },
+    "provider": {
+      "type": "oauth2",
+      "loginUrl": "${openmrsBase}/authenticationui/login/login.page",
+      "logoutUrl": "${openmrsBase}/authenticationui/login/login.page"
```

Comment:
Not really sure why this would be the logoutUrl?  Why wouldn't this be something like `/appui/header/logout.action`?

Reply:
Looking at the code, it seems like "logoutUrl" was more "the thing to link to after the OpenMRS logout is complete", which since we don't have any 2 factor place or elsewhere to log out, was just the login page.  However.  it is confusing, and there's no problem with logging out twice, I guess, I will switch to that if it works.